### DISCUSSION TITLE
selection box / block outline can be visible in multiple places

### DISCUSSION THREAD

**Discussion by LoganDark** [iPortalTeam/ImmersivePortalsMod#1606] [Non-Compatibility Issue Report]
Basically this

![image](https://user-images.githubusercontent.com/4723091/79931289-8c2ed500-83ff-11ea-93a1-c1b01b915103.png)

The hole on the left is a portal looking into the hole on the right

**Comment by qouteall** [comment]
This is not a problem.

**Comment by LoganDark** [comment]
I'm still considering making a fix for this. It should be pretty simple

**Comment by LoganDark** [comment]
![issue185](https://user-images.githubusercontent.com/4723091/80868054-deb49080-8c4c-11ea-9e97-22ac7e212b89.gif)
almost working :D
